unix_integration/resolver/Cargo.toml

```diff
@@ -110,3 +110,21 @@ kanidmd_testkit = { workspace = true }
 clap = { workspace = true, features = ["derive"] }
 clap_complete = { workspace = true }
 kanidm_build_profiles = { workspace = true }
+
+## Debian packaging
```

Comment:
Is this what powers cargo-deb ? Looks handy if it works 😄

Reply:
Yup! This is the appeal, much less bespoke pipeline and instead relying one someone else.